installation-linux.html

```diff
@@ -59,7 +59,9 @@
 
 <h3>Install dependencies</h3>
 
-<p>As Gregorio uses Lua<span class="tex">T<span class="epsilon">e</span>X</span> to produce the final printed socres, you need to obtain a <span class="tex">T<span class="epsilon">e</span>X</span> distribution which contains this engine.  We recommend <span class="tex">T<span class="epsilon">e</span>X</span>Live.  You can get this either via your package manager (in which case make sure you get the Lua<span class="tex">T<span class="epsilon">e</span>X</span> engine and tools, as package managers often split <span class="tex">T<span class="epsilon">e</span>X</span>Live up into multiple packages to reduce the download size) or from the <a href="https://www.tug.org/texlive/"><span class="tex">T<span class="epsilon">e</span>X</span>Live homepage.</a></p>
+<p>As Gregorio uses Lua<span class="tex">T<span class="epsilon">e</span>X</span> to produce the final printed socres, you need to obtain a <span class="tex">T<span class="epsilon">e</span>X</span> distribution which contains this engine.  We recommend <span class="tex">T<span class="epsilon">e</span>X</span>Live.  You can get this either via your package manager (in which case make sure you get the package which contains  the Lua<span class="tex">T<span class="epsilon">e</span>X</span> engine and tools, as package managers often split <span class="tex">T<span class="epsilon">e</span>X</span>Live up into multiple packages to reduce the download size) or from the <a href="https://www.tug.org/texlive/"><span class="tex">T<span class="epsilon">e</span>X</span>Live homepage.</a></p>
+
+<p>While we recommend that you get a complete <span class="tex">T<span class="epsilon">e</span>X</span>Live installation since it will provide you with the most flexibility going forward, we understand that not everyone has the disk space or time to install the roughly 4GB of material that this requires.  If you are trying to maintain a slimmer installation, you will need to ensure that you have the following packages installed: xcolor, kvoptions, ifluatex, graphicx, luatexbase, luaotfload, luamplib, and xstring.  Most of these packages should come with the Lua<span class="tex">T<span class="epsilon">e</span>X</span> engine, but at least xcolor and xstring are known to be in other collections on some platforms.</p>
```

Comment:
I've double checked the installed sizes of a complete TeXLive installation (all available packages installed), and they are roughly accurate (the Mac installation was actually a bit larger, so I upped that number a bit).  Merging.